va/drm/va_drm_auth_x11.c

```diff
@@ -115,7 +122,7 @@ drm_auth_x11_init(DRMAuthX11 *auth)
     if (!auth->display)
         return false;
 
-    auth->window = DefaultRootWindow(auth->display);
+    auth->window = None;
```

Comment:
Why this is equivalent to what was here before?

Reply:
It is unused, so for all intents and purposes we can make it anything that we want.